docs/checkbox.md

```diff
@@ -59,8 +66,8 @@ import { CheckBox } from 'react-native-elements'
 | onLongIconPress | none                                              | function               | onLongPress function for checkbox (optional)                                                                                                                                                                         |
 | onPress         | none                                              | function               | onPress function for container (optional)                                                                                                                                                                            |
 | onIconPress     | none                                              | function               | onPress function for checkbox (required)                                                                                                                                                                             |
-| checkedIcon     | check-square-o                                    | string                 | default checked icon ([Font Awesome Icon](http://fontawesome.io/icons/)) (optional)                                                                                                                                  |
-| uncheckedIcon   | square-o                                          | string                 | default checked icon ([Font Awesome Icon](http://fontawesome.io/icons/)) (optional)                                                                                                                                  |
+| checkedIcon     | check-square-o                                    | string | React Native Component                 | default checked icon ([Font Awesome Icon](http://fontawesome.io/icons/)) (optional)                                                                                                                                  |
+| uncheckedIcon   | square-o                                          | string | React Native Component                 | default checked icon ([Font Awesome Icon](http://fontawesome.io/icons/)) (optional)                                                                                                                                  |
```

Comment:
That looks weird because of the `|`, the description was kicked out lol
![image](https://user-images.githubusercontent.com/17592779/37511002-7220233a-28fd-11e8-8a84-fbd878ab1322.png)

Reply:
Lol 😲 good catch